@vates/fatfs/helpers.js

```diff
@@ -141,17 +141,17 @@ exports.checksumName = function (buf,off) {
 
 /* comparing C rounding trick from FAT spec with Math.ceil
 function tryBoth(d) {
-    var a = ((D.RootEntCnt * 32) + (D.BytsPerSec - 1)) / D.BytsPerSec >>> 0,
-        b = Math.ceil((D.RootEntCnt * 32) / D.BytsPerSec);
```

Comment:
It's very hard for me to determine if commented code reference typos in external API or if they're just typos...